sample_config.json

```diff
@@ -7,7 +7,8 @@
 	"ignored_ports": [ 123 ]
     },
     "countries": {
-       "whitelist": ["IT", "DE", "CH", "NL"],
+       "whitelist": ["IT", "DE", "CH", "NL",
+                     "AF", "AS", "EU", "NA", "OC", "SA", "AN"],
```

Comment:
How do you plan to specify the continent in the config file? I don't think it's wise to mix countries with continents, what do you think?

Reply:
Yes, it's true. I should specify continents in a separate json object "continents"